db/seeds/run-seed.js

```diff
@@ -1,6 +1,6 @@
 const devData = require('../data/development-data/index.js');
 const seed = require('./seed.js');
-const db = require('../connection.js');
+const db = require('../index.js');
 
 const runSeed = () => {
   return seed(devData).then(() => db.end());
```

Comment:
ideally we wanted to be seeding before each test (see the notes on seeding with pg for a refresher!)